src/main/java/de/paginagmbh/epubchecker/PaginaEPUBChecker.java

```diff
@@ -19,17 +19,17 @@
  *
  * @author      Tobias Fischer
  * @copyright   pagina GmbH, Tübingen
- * @version     2.0.2
- * @date        2020-07-14
+ * @version     2.0.3
+ * @date        2021-01-07
  */
 public class PaginaEPUBChecker {
 
 	// +++++++++++++++++++++++++ DON'T FORGET TO UPDATE EVERYTIME ++++++++++++++++++ //	
 
-	public static final String PROGRAMVERSION = "2.0.2";
-	public static final String VERSIONDATE = "14.07.2020";
-	public static final String PROGRAMRELEASE = "";	// "" or "Beta"
-	public static final String RELEASENOTES = "- Update the official W3C EPUBCheck library to the latest release v4.2.4";
+	public static final String PROGRAMVERSION = "2.0.3";
+	public static final String VERSIONDATE = "07.01.2021";
+	public static final String PROGRAMRELEASE = "Beta";	// "" or "Beta"
```

Comment:
@Sannwama @sermo-de-arboribus oh, das ist nicht gut! Das hätte vor dem Release entfernt werden müssen. Wenn ihr die Binaries in diesem Zustand ausliefert, dann werden beim End-User die nächsten Updates nicht mehr funktionieren weil er immer nach Beta-Updates sucht.